scripts/lib/cluster.sh

```diff
@@ -22,24 +22,51 @@ function up-test-cluster() {
     fi
 
     echo -n "Configuring cluster $CLUSTER_NAME"
-    AWS_K8S_TESTER_EKS_NAME=$CLUSTER_NAME \
-        AWS_K8S_TESTER_EKS_LOG_COLOR=true \
-        AWS_K8S_TESTER_EKS_KUBECONFIG_PATH=$KUBECONFIG_PATH \
-        AWS_K8S_TESTER_EKS_KUBECTL_PATH=$KUBECTL_PATH \
-        AWS_K8S_TESTER_EKS_S3_BUCKET_NAME=$S3_BUCKET_NAME \
-        AWS_K8S_TESTER_EKS_S3_BUCKET_CREATE=$S3_BUCKET_CREATE \
-        AWS_K8S_TESTER_EKS_PARAMETERS_VERSION=${K8S_VERSION%.*} \
-        AWS_K8S_TESTER_EKS_PARAMETERS_ENCRYPTION_CMK_CREATE=false \
-        AWS_K8S_TESTER_EKS_PARAMETERS_ROLE_CREATE=$ROLE_CREATE \
-        AWS_K8S_TESTER_EKS_PARAMETERS_ROLE_ARN=$ROLE_ARN \
-        AWS_K8S_TESTER_EKS_ADD_ON_MANAGED_NODE_GROUPS_ENABLE=true \
-        AWS_K8S_TESTER_EKS_ADD_ON_MANAGED_NODE_GROUPS_ROLE_CREATE=$ROLE_CREATE \
-        AWS_K8S_TESTER_EKS_ADD_ON_MANAGED_NODE_GROUPS_ROLE_ARN=$MNG_ROLE_ARN \
-        AWS_K8S_TESTER_EKS_ADD_ON_MANAGED_NODE_GROUPS_MNGS=$MNGS \
-        AWS_K8S_TESTER_EKS_ADD_ON_MANAGED_NODE_GROUPS_FETCH_LOGS=true \
-        AWS_K8S_TESTER_EKS_ADD_ON_NLB_HELLO_WORLD_ENABLE=$RUN_TESTER_LB_ADDONS \
-        AWS_K8S_TESTER_EKS_ADD_ON_ALB_2048_ENABLE=$RUN_TESTER_LB_ADDONS \
-        $TESTER_PATH eks create config --path $CLUSTER_CONFIG 1>&2
+
+    echo "Custom networking - $RUN_CUSTOM_NETWORKING_TESTS"
+
+    if [[ "$RUN_CUSTOM_NETWORKING_TESTS" == true ]]; then
+        #maxPods = (15 - 1) * (50 - 1) + 2 = 688
```

Comment:
leftover?

Reply:
Yeah I just added the comment, this should be replaced with describe instance to get max ip and eni limit.

Reply:
I will make it dynamic in the later PRs, will discuss with you offline.